### PR TITLE
Add bow and docfreq extracted from PGA

### PR DESCRIPTION
**Pull request by r0mainK**
All in title, tell me if you think some things should be changed @vmarkovtsev @campoy @marnovo

### CODE REVIEW

File: bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# bow
```
Comment (campoy) [inline bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md]: what is bow? bag of words? an actual bow? balls of wonder?

File: bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Bags of features, extracted in july 2018 from 7.8 million distinct files from PGA (taking only the `HEAD` commit), using all implemented extractors in `sourced.ml` at the time (`identifiers`, `literals`, `graphlets`, `children`, `node2vec` and `uast2seq`) and all languages parsable by Babelfish (Go, Java, Python, Bash, JavaScript and Ruby). This was done to try to use [apollo](https://github.com/src-d/apollo) at scale. We hit `scipy.sparse` limits while trying to merge sparse matrices for all bags, so this is only one of three `BOW` model holding bags.
```
Comment (campoy) [inline bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md:3]: `BOW` or `bow`?

File: bow/694c20a0-9b96-4444-80ae-f2fa5bd1395b.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# bow
```
Comment (campoy) [inline bow/694c20a0-9b96-4444-80ae-f2fa5bd1395b.md]: Why is this 3 times?
They all have the same description and version 1.0.0 but different sizes :-/

**Review by campoy — CHANGES_REQUESTED** [review]

File: bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md (hunk 2 above)
Comment (vmarkovtsev) [reply bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md:3]: Should be BOW everywhere. I think we lack proper, long names for the models and use their type ids for the headers.

File: bow/694c20a0-9b96-4444-80ae-f2fa5bd1395b.md (hunk 3 above)
Comment (r0mainK) [reply bow/694c20a0-9b96-4444-80ae-f2fa5bd1395b.md]:
```
We hit `scipy.sparse` limits while trying to merge sparse matrices for all bags, so this is only one of three `BOW` model holding bags.
```

From the description

**Pull request merged**